tools/ansible/tasks/wsu/main.yaml

```diff
@@ -0,0 +1,54 @@
+---
+- hosts: localhost
+  vars:
+    project_root: "{{ playbook_dir }}/../../../.."
+    kubelet_location: "https://dl.k8s.io/v1.14.0/kubernetes-node-windows-amd64.tar.gz"
+    tmp_path: "/tmp/wsu"
+
+  tasks:
+    - name: Gather required files
+      block:
+
+        - name: Create directory to store files to be used later
+          file:
+            path: "{{ tmp_path }}"
+            state: directory
+
+        - name: Build WMCB
+          make:
+            target: build
+            chdir: "{{ project_root }}"
+        - name: Move WMCB to temporary directory
+          command: mv "{{ project_root }}/wmcb.exe" "{{ tmp_path }}/wmcb.exe"
+
+        - name: Download Windows node kubelet
+          get_url:
+            url: "{{ kubelet_location }}"
+            dest: "{{ tmp_path }}/kube.tar.gz"
+
+        - name: Extract kubelet
+          unarchive:
+            src: "{{ tmp_path }}/kube.tar.gz"
+            dest: "{{ tmp_path }}"
+
+        - name: Grab kubelet from extracted directories
+          copy:
+            src: "./tmp/kubernetes/node/bin/kubelet.exe"
+            dest: "./tmp/kubelet.exe"
+
+- hosts: win
+  vars:
+    tmp_path: "{{ playbook_dir }}/tmp"
```

Comment:
Why create a new variable here? Do you forsee any use of this?

Reply:
We need to recreate the variable for each host.

Reply:
I was mistaken, changed this.